workflow/controller/operator.go

```diff
@@ -2977,6 +2978,26 @@ func (woc *wfOperationCtx) executeSuspend(nodeName string, templateScope string,
 	return node, nil
 }
 
+func resolveInputFieldsForSuspendTemplate(tmpl *wfv1.Template) {
```

Comment:
I don't really understand how this works, can you explain?

Reply:
I convert the input string `{"enum": ["one", "two", "three"]}` to an actual enum object in template